ckanext/scheming/templates/scheming/display_snippets/datetime.html

```diff
@@ -0,0 +1 @@
+{{ data[field.field_name].split()[0]+'T'+data[field.field_name].split()[1] }}
```

Comment:
Not really no, but it didn't occur it was our client's preference. I can make it generic and override it in our project